contracts/infrastructure_0.5/base/Managed.sol

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018  Argent Labs Ltd. <https://argent.xyz>
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.5.4;
+
+import "../../infrastructure/base/Owned.sol";
+
+/**
+ * @title Managed
+ * @notice Basic contract that defines a set of managers. Only the owner can add/remove managers.
+ * @author Julien Niset - <julien@argent.xyz>
+ */
+contract Managed is Owned {
```

Comment:
Why is this added? I can't see a difference between this and the parallel instance in `infrastructure/base`?

Reply:
the 0.5 compiler doesn't understand `virtual` so this file is the same as the one in `infrastructure/base` but without the `virtual`.

Reply:
Okay then maybe update the pragma in `infrastructure` from `pragma solidity >=0.5.4 <0.7.0;` to `pragma solidity ^0.6.12;` might as well do that to `Owned` in there as well for consistency.

Reply:
Ok for `infrastructure/base/Managed.sol` but why change `Owned.sol` ? The same file can still be used by both `0.5` and `0.6`.